hReload.py

```diff
@@ -79,19 +81,20 @@ def check_for_new_commit(scheduler, pHandle, shaPr):
 
 
 
+
 # Setup scheduler and schedule the commit check
 my_scheduler = sched.scheduler(time.time, time.sleep)
 my_scheduler.enter(float(env_vars["COMMIT_CHECK_INTERVAL"]), 1, check_for_new_commit, (my_scheduler, pHandle, shaPr))
 my_scheduler.run()
 
-#while (True):
```

Comment:
All of this commented code should go, I'll take care of it in the eventual refactor.